functions/Get-DbaDbMailProfile.ps1

```diff
@@ -62,6 +62,12 @@ function Get-DbaDbMailProfile {
 
         Returns the DBMail profiles for "sql2014","sql2016" and "sqlcluster\sharepoint"
 
+    .EXAMPLE
+        PS C:\> $servers = "sql2014","sql2016", "sqlcluster\sharepoint"
```

Comment:
Maybe give it a space after the first comma? Same in the example you copied the code from.

Reply:
Added, although this is a broad issue and could be covered by separate PR for other functions.